### PR TITLE
VACMS 3509: Video embeds for CMS Help Center pages

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #3509 .

## Testing done
Manual tests so far.  I was thinking about adding Behat tests...

## Screenshots
Screen Shot 2020-12-21 at 4 15 31 PM
Screen Shot 2020-12-21 at 4 15 23 PM

## QA steps
As a user with the ability to view and/or create CMS Help Page nodes:
1. Create or edit a node so that it has video media in the Main Content 
1. Then validate Acceptance Criteria from issue #3509:
    - [x] Media:Video has a new view mode enabled "User Guides"
    - [x] Media:Video "User Guides" view mode  is configured as follows:
        * displayed fields:
           * Video URL | Label - hidden | format - Video - embed video responsive
           * Name | Label - hidden | format - plain text
           * Duration | Label - inline | format - Natural human languages + remove the `and` separator
           * Description | Label hidden | Plain text
    - [x] Paragraph "Media list- videos" has a new view mode enabled "User guides" w/ following settings:
        * Section Header: Label - hidden; format - plain text
        * Videos: Label hidden; format - Rendered Entity w/ view mode - User Guides
    - [x] new template is added for `paragraph--media-list-videos--user-guides.html.twig` that renders field_section_header as an `h2` heading
    - [x] CMS Help Page content type - field_content_block `/admin/structure/types/manage/documentation_page/fields/node.documentation_page.field_content_block` allows `Media list - Videos` paragraph type as an option

### Definition of Done
- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ndouglas** [comment]
@oksana-c @rachel-kauff [Here](https://pr3845-bjte00jbyxkzt6e6z8ghrmbwpfynf3vs.ci.cms.va.gov/node/6884/latest) is a preview that hopefully contains enough information to review.  I added the duration and description fields for the two media items.

File: docroot/themes/custom/vagovadmin/templates/paragraph/paragraph--downloadable-file--user-guides.html.twig [hunk 1]
```diff
@@ -46,7 +46,7 @@
       {% set file_extension = media_object.link_url | split('.') | last | upper %}
       <a {{ media_object.attributes }} target="_blank" href="{{ media_object.link_url }}" download="{{ media_object.link_url }}" aria-label="{{ 'Download' | trans }}">{{ media_object.link_title }} ({{ file_extension }})</a>
     {% elseif media_object.bundle is same as("video") %}
-      <a {{ media_object.attributes }} target="_blank" href="{{ media_object.link_url }}">{{ media_object.link_title }}</a>
+      {{ content.field_media }}
```
Comment (oksana-c) [inline docroot/themes/custom/vagovadmin/templates/paragraph/paragraph--downloadable-file--user-guides.html.twig:49]: we are not changing downloadable file paragraph config in this PR. The only paragraph type affected by ACs is "Media list - Videos" in User Guides view mode. apologies if that wasn't clear.

**Review by oksana-c — CHANGES_REQUESTED** [review]
video list layout looks great! I left a comment re: downloadable file paragraph changes

duration field label could use some styling. I pasted some scss below (warning: untested code). The result should look as shown on the screenshot.
Let's make sure that we're only targeting the duration field label on a media list paragraph in user guides view mode.

file to add scss - docroot/themes/custom/vagovadmin/assets/scss/user_guides.scss

#block-vagovadmin-content {
  .paragraph--type--media-list-videos {
    &.paragraph--view-mode--user-guides {
      .field--name-field-duration {
        .field__label {
          font-size: 16px;
          text-transform: capitalize;
        }
      }
    }
  }
}

Screen Shot 2020-12-22 at 10 19 21 AM

File: docroot/themes/custom/vagovadmin/templates/paragraph/paragraph--downloadable-file--user-guides.html.twig (hunk 1 above)
Comment (ndouglas) [reply docroot/themes/custom/vagovadmin/templates/paragraph/paragraph--downloadable-file--user-guides.html.twig:49]: Ah, okay.  Not a problem 😄

**Comment by oksana-c** [comment]
added minor styling tweaks. here's the result.
✅ good to merge once we get thumbs up from @rachel-kauff and the build is green


Screen Shot 2020-12-22 at 6 01 02 PM

**Review by oksana-c — APPROVED** [review]

**Comment by ndouglas** [comment]
Tugboat hasn't been building the previews for this PR automatically, for some reason, but it should be current as of now.

**Comment by rachel-kauff** [comment]
@ndouglas cc: @oksana-c Just reviewed, looks great. Thank you so much for your work on this!

**Pull request merged**